src/code42cli/cmds/auditlogs.py

```diff
@@ -0,0 +1,255 @@
+    "-c",
+    "--use-checkpoint",
+    help="Only get audit-log events that were not previously retrieved."
+)
+
+
+@click.group(cls=OrderedGroup)
+@sdk_options(hidden=True)
+def audit_logs(state):
+    """Retrieve audit logs."""
+    # store cursor getter on the group state so shared --begin option can use it in validation
+    state.cursor_getter = _get_audit_log_cursor_store
+
+
+@audit_logs.command()
+@click.argument("checkpoint-name")
+@sdk_options()
+def clear_checkpoint(state, checkpoint_name):
+    """Remove the saved file event checkpoint from `--use-checkpoint/-c` mode."""
+    _get_audit_log_cursor_store(state.profile.name).delete(checkpoint_name)
+
+
+@audit_logs.command()
+@filter_options
+@format_option
+@checkpoint_option
+@sdk_options()
+def search(
+    state,
+    begin,
+    end,
+    event_type,
+    username,
+    user_id,
+    user_ip,
+    affected_user_id,
+    affected_username,
+    format,
+    use_checkpoint,
+):
+    """Search audit logs."""
+    save_checkpoint = None
+    if use_checkpoint:
+        cursor = _get_audit_log_cursor_store(state.profile.name)
+        save_checkpoint = lambda ts: cursor.replace(use_checkpoint, ts)
+        begin = cursor.get(use_checkpoint)
+    
+    _search(
+        state.sdk,
+        format,
+        save_checkpoint,
+        begin_time=begin,
+        end_time=end,
+        event_types=event_type,
+        usernames=username,
+        user_ids=user_id,
+        user_ip_addresses=user_ip,
+        affected_user_ids=affected_user_id,
+        affected_usernames=affected_username,
+    )
+
+
+@audit_logs.command()
+@filter_options
+@checkpoint_option
+@server_options
+@send_to_format_options
+@sdk_options()
+def send_to(
+    state,
+    hostname,
+    protocol,
+    format,
+    begin,
+    end,
+    event_type,
+    username,
+    user_id,
+    user_ip,
+    affected_user_id,
+    affected_username,
+    use_checkpoint,
+):
+    """Send audit logs to the given server address."""
+    save_checkpoint = None
+    if use_checkpoint:
+        cursor = _get_audit_log_cursor_store(state.profile.name)
+        save_checkpoint = lambda ts: cursor.replace(use_checkpoint, ts)
+        if not begin:
+            begin = cursor.get(use_checkpoint)
+    
+    _send_to(
+        state.sdk,
+        hostname,
+        protocol,
+        format,
+        save_checkpoint,
+        begin_time=begin,
+        end_time=end,
+        event_types=event_type,
+        usernames=username,
+        user_ids=user_id,
+        user_ip_addresses=user_ip,
+        affected_user_ids=affected_user_id,
+        affected_usernames=affected_username,
+    )
+
+
+def _search(sdk, format, save_checkpoint, **filter_args):
+    formatter = OutputFormatter(format, AUDIT_LOGS_DEFAULT_HEADER)
+    response_gen = sdk.auditlogs.get_all(**filter_args)
+    events = _get_all_audit_log_events(response_gen)
+    event_count = len(events)
+    if not event_count:
+        click.echo("No results found.")
```

Comment:
`click.echo("No results found.")` is used like 4 times throughout the codebase, might be worth extracting into a shared method or a constant.